cmake/FindICONV.cmake

```diff
@@ -4,7 +4,7 @@
 # ICONV_LIBRARIES
 #
 
-if (TARGET_OS_LINUX)
+if (TARGET_OS_FREEBSD OR TARGET_OS_LINUX)
```

Comment:
I asked from @kbelyavs and he answers this is no-op on Linux, because Linux has iconv_* functions in libc. But I see the same on FreeBSD 11.2. Maybe FreeBSD 10 differs, I will check.

Reply:
FreeBSD 10.4 contains iconv functions in libc too.

Reply:
Thanks for pointing out 16a636f and dcac64a. I meant to build tarantool with iconv from base system (libc), thus I added the patch. Since upstream decided to use libiconv from ports, the patch is not needed.

BTW, FreeBSD 10 is expired on Oct 31.

Reply:
We need to elaborate what are the incompatibilities that was stated in 16a636f29b98a986da902cbf81b8ca9cf3fa1214 and decide finally. Filed #3791 to discuss it further. Please, participate if you have some details about the problems with base iconv or any other details. I don't use FreeBSD personally, so can miss primer approaches.

Reply:
BTW, checked 12.0-BETA-3: built and run tarantool from source successfully.